### DISCUSSION TITLE
Cannot connect to networked drive

### DISCUSSION THREAD

**Discussion by speedreader** [rern/rAudio#1260] [Fixed Issues]
rAudio 20230212
Raspberry Pi 3B 

I've got a USB drive mounted to my Apple AirportExtreme. I've been running the old Rune Audio on my RPi and have had no problems connecting to the drive. Upgrading to newest rAudio as I'm want run two systems on the same network.

I cannot get rAudio to find the drive no matter what settings I try. The drive is called 'Beverage_Lizard' and the folder I want to access is 'Muzak_LIbrary'. 

<img width="548" alt="Screen Shot 2023-03-04 at 10 45 38 AM" src="https://user-images.githubusercontent.com/16739809/222919345-f67a2f05-1b54-4664-9334-311c6c5aeac3.png">

As others have noted I cannot type either a \ or / into the field for 'Share name' so I can't put the file path to the music library itself. Not great, but okay. But even if I just use the whole drive name for the share path I still get the following error. Every other iteration I could think up results in the same message.

<img width="520" alt="Screen Shot 2023-03-04 at 10 32 57 AM" src="https://user-images.githubusercontent.com/16739809/222919355-ca198a22-88ae-41e7-a38e-d9d4f4a5ab62.png">

Any thoughts on what's happening here? I keep thinking there's something at router but, as I said, the older Rune Audio had no problems finding it at //10.0.1.1/Beverage_Lizard/Muzak_Library

**Comment by speedreader** [comment]
FWIW: Upgraded to the release that was just posted (rAudio 20230304). No difference.

**Comment by R101G** [comment]
I made a similar mistake recently. You just need to enter the share name, exactly as it appears in the /etc/samba/smb.conf file.

**Reply by speedreader** [reply]
Thanks R101G... 

Two things. I'm not seeing any network drive paths in my /etc/samba/smb.conf file. 

Also, as I and others have noted in some other threads, the 'Share name' field simply won't accept either "/" or "\" characters. They just disappear when you type them. So I can't type any file path there. Is SSH'ing in the only surefire way to do this now?

Also, also (so, technically three things...) I have SMB enabled. Why is it showing 'inactive (dead)'?

<img width="688" alt="Screen Shot 2023-03-04 at 1 26 38 PM" src="https://user-images.githubusercontent.com/16739809/222925125-f390c2c8-06c3-40a5-b59d-a8dc608e5da2.png">

Thanks again

**Reply by R101G** [reply]
Is it turned on in features?

**Reply by speedreader** [reply]
Yep.

**Reply by R101G** [reply]
The output you posted does not look like the result of systemctl status smb from rAudio?

**Comment by R101G** [comment]
Have you added the storage drive on the System menu page? Does it mount?

**Reply by speedreader** [reply]
Hahaha... I guess I wasn't clear. This is the problem I'm having: I can't get the storage device added when I try from the System page. That's where the screenshots are from. Also my NAS isn't a NAS per se. It's just a 2.5" external USB drive. 

Again, this all worked fine with the old RunePlayer sofware.

**Reply by R101G** [reply]
OK, I looked it up! It's a router with a USB port. So it's very likely to need the vers=2.0 option, and you probably don't have ssh access to it. If Samba is turned on in the rAudio features menu, rAudio should just connect to the share name set up on the router. The router might treat the USB stick as the share name, like my Asus does.

**Comment by R101G** [comment]
Another thought. I don't know anything about the NAS you have, but if it's old, it might not handle Samba v3, so try putting vers=2.0 in the options field. Also, of course, the smb.conf file will be in a different location on your NAS.
